test/mlir/conversion/onnx_to_tosa/Tensor/Reshape.mlir

```diff
@@ -25,5 +38,9 @@ func.func @test_shape_zero_reshape(%arg0: tensor<32x512x1x1xf32>) -> tensor<32x5
   %0 = "onnx.Constant"() {value = dense<[32, 0]> : tensor<2xi64>} : () -> tensor<2xi64>
   %1 = "onnx.Reshape"(%arg0, %0) : (tensor<32x512x1x1xf32>, tensor<2xi64>) -> tensor<32x512xf32>
   return %1 : tensor<32x512xf32>
-  //CHECK-LABEL: "onnx.Reshape"
+// CHECK-LABEL:   func.func @test_shape_zero_reshape(
+// CHECK-SAME:                                       %[[VAL_0:.*]]: tensor<32x512x1x1xf32>) -> tensor<32x512xf32> {
+// CHECK:           %[[VAL_1:.*]] = "tosa.const"() {value = dense<[32, 0]> : tensor<2xi64>} : () -> tensor<2xi64>
```

Comment:
This check doesn't seem relevant for the reshape rewrite - please remove if it isn't.